certbot/certbot/tests/util.py

```diff
@@ -36,6 +26,18 @@
 from certbot.compat import os
 from certbot.display import util as display_util
 
+try:
```

Comment:
I think this is a consequence of executing `isort` (I got that serveral time). Did you have the intention to move this import there?

Reply:
You're right that this was due to `isort`. I tried to explain this in my initial post when I said:

> The extra moving around of imports is due to me using `isort` to help me keep dependencies in sorted order after replacing imports of `six`.

I can change them back if you want, however, I personally don't mind the change and think we may as well sort imports as done by `isort` to make it easier for us to use the tool.

Reply:
In fact, it superiorily annoys me each time I need to manually fix the imports on that matter each time I run isort, so I am fine with you keeping the order as it is in the PR.

Anyway things will be back ti normal once we remove the `mock` dependency and do not need to do a try/except on it anymore.

Reply:
> In fact, it superiorily annoys me each time I need to manually fix the imports on that matter each time I run isort,

That'd annoy me too! In general if there are things on Certbot that are making your life harder, by all means flag/ask about them. Let's not make things harder than they need to be.